backend/app/use_cases/__init__.py

```diff
@@ -1,9 +1,9 @@
+from .change_password_interactor import ChangePasswordInteractor
 from .generate import Generate
 from .get_headers import GetHeaders
 from .get_last_login_data import GetLastLoginData
 from .get_values_under_header import GetValuesUnderHeader
-from .upload_data import UploadData
-from .share import Share
 from .login_user_interactor import LoginUserInteractor
 from .register_user_interactor import RegisterUserInteractor
-from .change_password_interactor import ChangePasswordInteractor
```

Comment:
Why you removed this line

Reply:
I used pylint to fix import statements and ran the package "isort" that fixes the import statements, so this was done automatically. All tests still pass so shouldn't be a probelm